dist/multiselect.js

```diff
@@ -174,6 +175,7 @@
          * @param {Object} ngModelController
          */
         function link(parentScope, element, attrs, ngModelController) {
+            var _conjunctionText = getSettingValue('conjunctionText');
```

Comment:
Not really familiar with how this relates to values set in attrs

other than that this seems great 👍

Reply:
The implementation of this method essentially checks for an attribute first and then falls back on the globally configured value:

https://github.com/namoscato/angular-multiselect/blob/73166143189f0c95c23bba10c9d8630e1667c38a/src/lib/multiselect.directive.js#L203-L216